bindings/python/src/utils/normalization.rs

```diff
@@ -44,6 +44,15 @@ impl From<PyPattern<'_>> for tk::normalizers::replace::ReplacePattern {
     }
 }
 
+impl From<PyPattern<'_>> for tk::pre_tokenizers::split::SplitPattern {
```

Comment:
is it fine that `PyPattern` is implemented for `pre_tokenizers` in this file or should I move this to `utils/pretokenization.rs` ?

Reply:
Yes, I think it's the best place to have it. I like having the `From` defined in the same place that one of the involved types, so here just after `PyPattern` seems great.
I see you finally moved it to `utils/pretokenization.rs` so I'll move it back here.